src/components/node-list/node-list-items.js

```diff
@@ -149,8 +149,8 @@ export const getFilteredModularPipelines = createSelector(
  * @return {array} Node list items
  */
 export const getFilteredModularPipelineItems = createSelector(
```

Comment:
Hey, I just realised that node-list-items contains a lot of selectors. Shouldn't selectors be in the selectors folder of the project and imported to components as required. Why is it not this case here :S

Reply:
yes, this is indeed a problem that had been raised multiple times - this has been set up historically. `node-list-items` are the files that is used to contain all selectors related to the sidebar. 

The main reason behind this seperation, I believe, is that this set of selectors:
a) are uniquely arranged just for the use of the node-list component rather than pertaining the selection of a more 'general' data type ( tags, modular-pipeline, etc)
b) the selectors feed in from a 'state' that is an object passed in from node-list rather than from the global state object

I also had the same confusion as to why this is arranged here, however on having developed extensively on the sidebar I do find that this arrangement makes sense. You can treat this as a series of util functions of the sidebar that makes uses of the selector setup to allow constant updates for all of its related selectors. 

FYI - In terms of this question itself,  this is out of scope of this PR and would be better raised in a seperate occasion.

Reply:
Thanks for the clarification Susanna. 
I think we should set norms on raising questions in and out or PR because I always get asked questions that are outside the scope of the project in my PR's as well. Maybe next retro.